DOCS-3871: Add camera streaming info (#4233)
viamrobotics/docs: docs/dev/tools/common-errors.md

See [Module Configuration Details](/operate/reference/module-configuration/#environment-variables) for more information on these environment variables.

## Common warnings

### Unable to create PeerConnection with module

**Full Warning:** `Unable to create PeerConnection with module. Ignoring.`

**Description:** Indicates that while the gRPC connection to the module is working as expected, the connection to the module does not support efficient video streaming over WebRTC.
Only some Go-based camera modules support optimized video streaming over WebRTC.

{{% hiddencontent %}}

> Any Viam SDK can be used to write a camera module, but only Go-based modules can implement optimized video streaming over WebRTC.

Reviewer: ```suggestion
You can use any Viam SDK to implement a camera module, but only Go-based modules can access optimized video streaming over WebRTC.
```
[opt] this is hidden content so it's going to wind up filtered through AI anyway, but i'm a sucker for user-centric language. take it or leave it!

Author: Ah you’re right! I haven’t been as thoughtful in my hidden content verbiage but I suppose might as well
